.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,35 @@
+<!--- Please provide a general summary of your changes in the Title above -->
+
+# PR Description
+
+<!--- Describe the changes your PR makes in detail -->
+
+References issue: # (ISSUE)
+
+## Motivation and Context
+
+<!--- Why is this change required? What problem does it solve? -->
+<!--- If it fixes an open issue, please link to the issue here. -->
+
+## Tests and Coverage
+
+<!--- Please describe in detail how you tested your changes. -->
+<!--- Include details of your testing environment, tests ran to see how your change affects other areas of the code, etc. -->
+
+## Types of changes
```

Comment:
Aren't we essentially replicating the PR Labels already built in to GitHub here?

Reply:
I these have a little bit of over lap but still are okay on their own, I think it's good to have it in the PR's written documentation rather than just hope that the correct label has been applied aswell.

Reply:
Amm... is it only a bit of overlap though? 😅

These are the pairs I see here between the tickbox and a label

Bug fix - Bug
New feature - Enhancement (arguably we could rename this if we feel 'New Feature' sounds more descriptive - I kinda agree)
Docs - Documentation

The only one that we don't have a label for yet is 'Breaking change'. That's 3/4 of them already being a label, we could easily add 'Breaking Change' as another one. What benefits do you see of having these as tickboxes as opposed to labels? I feel that labels are more useful as they show on the list of Issues or PRs, making it easier to browse the respective history in the future and search for specific labels - while these tickboxes would be hidden in the PR content.

Also - if we're 'hoping' they'd put the right label on (which actually I think we should be doing - not them - as we understand the scope of the project better), we'd have to 'hope' that they'd put the X in the right tickbox just as much, wouldn't we? In my experience with GitHub I've seen a ton of people completely or partially ignore the templates - as much as I wished it was otherwise!

Reply:
Yep you're right. We'll stick with the labels, what might be nice to do for this section then is to ask "Have you assigned the appropriate Github labels to this PR" seems to be the best of both worlds.

Reply:
Cool, thanks for reviewing these points 👍 Yup that sounds like a good tickbox to have on the list 😊